.claude/plugins/interop-docs/skills/docs-verification/references/verification-protocol.md

```
@@ -0,0 +1,228 @@
+```typescript
+const result = await someFunction();
+```
+
+### Output Blocks
+Use separate code block:
+```typescript
+// Code
+const result = await getAddress("vitalik.eth");
+```
+Output:
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)